.env.default

```
@@ -1,3 +1,12 @@
-MYSQL_PASSWORD=
\ No newline at end of file
+MYSQL_PASSWORD=
+
+KAKAO_CLIENT_ID=
+KAKAO_CLIENT_SECRET=
+
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+
+NAVER_CLIENT_ID=
+NAVER_CLIENT_SECRET=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)